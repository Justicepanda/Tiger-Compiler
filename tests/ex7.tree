$
	<tiger-program>
		let
		<declaration-segment>
			<type-declaration-list>
				<type-declaration>
					type
					id
