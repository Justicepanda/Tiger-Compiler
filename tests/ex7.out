LET TYPE ID ASSIGN
Parsing error (line 2):         type int_arr :=  <-- ":=" is not a valid token. Expected "=".
Tiger-Compiler: Compilation unsuccessful.